web/theme.dart

```diff
@@ -0,0 +1,63 @@
+import 'dart:html';
+
+void init() {
+  var theme = document.getElementById('theme') as InputElement;
+
+  var elementB = document.body;
+
+  theme.addEventListener('change', (event) {
+    if (theme.checked == true) {
+      if (theme.getAttribute('value') == 'light-theme') {
+        elementB?.classes.remove('light-theme');
+        elementB?.classes.toggle('dark-theme');
+        theme.setAttribute('value', 'dark-theme');
+      } else {
+        elementB?.classes.remove('light-theme');
+        elementB?.classes.toggle('dark-theme');
+        theme.setAttribute('value', 'dark-theme');
+      }
+      window.localStorage['checked'] = 'true';
```

Comment:
Should the name of this variable refer to 'theme' in some way? It seems weird to just have a variable in the `localStorage` map called "checked."